public/index.html

```diff
@@ -3,6 +3,10 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link href="https://makasiini.hel.ninja/delivery/HelsinkiGrotesk/533af26cf28d7660f24c2884d3c27eac.woff" rel="preload" as="font" type="font/woff" crossorigin="anonymous" />
+    <link href="https://makasiini.hel.ninja/delivery/HelsinkiGrotesk/7c46f288e8133b87e6b12b45dac71865.woff" rel="preload" as="font" type="font/woff" crossorigin="anonymous" />
+    <link href="https://makasiini.hel.ninja/delivery/HelsinkiGrotesk/565d73a693abe0776c801607ac28f0bf.woff" rel="preload" as="font" type="font/woff" crossorigin="anonymous" />
+
```

Comment:
For what are these?

Reply:
These are for preloading fonts as suggested by https://developers.google.com/speed/pagespeed/insights/?hl=fi&url=https%3A%2F%2Flinkedcomponents-ui.test.kuva.hel.ninja%2Ffi&tab=desktop . I guess the reason is to avoid Flash of Unstyled Text situation. More info can be read for example here: https://wp-rocket.me/blog/font-preloading-best-practices/